packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -45,6 +45,7 @@ export const messages = {
   parameter_gatherer_enter_dir_name:
     'Enter desired directory (Press Enter to confirm or Esc to cancel)',
   parameter_gatherer_enter_username_name: 'Enter target username',
+  parameter_gatherer_enter_alias_name: 'Enter an alias',
```

Comment:
@ruthemmanuelle may you verify that this text is ok?

Reply:
Maybe 'Enter a scratch org alias', just to be 100% clear?